core/frontend/src/components/parameter-editor/InlineParameterEditor.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <v-form
```

Comment:
I don't quite recall what the issue was. care to elaborate?
also it looks like we use forms quite a lot? is this a different situation?

Reply:
> I don't quite recall what the issue was. care to elaborate? also it looks like we use forms quite a lot? is this a different situation? (first result in google is purple but I don't recall what happened)

https://github.com/bluerobotics/BlueOS/issues?q=is%3Aissue+NS_BINDING_ABORTED+is%3Aclosed

Reply:
and: https://github.com/bluerobotics/BlueOS/pull/2351/files

Reply:
ok that is weird. It must be a combination of things, though, right? maybe some request initialized in the form and then aborted because of the submit action?
We do have 15 other v-forms in the code, and I'm actively relying on the form validation in this component.
I'm not looking forward to changing it if the issue isn't really there.
@joaoantoniocardoso can you try the image plz? I'll do a bunch of testing here too.

Reply:
I would be okay if @joaoantoniocardoso fails to replicate the issue, he's pretty good on that :laughing: